### ISSUE TITLE
Initialize jnumpy in a new process could be invalid

### ISSUE THREAD

**Issue by songjhaha** [Suzhou-Tongyuan/jnumpy#60]
After #59 merged, ~jnumpy throws error if `CF_TYPY_PY_APIPTR` is invalid by checking if the environment variable `CF_TYPY_PID` equals to the identity of the current process(#42). So if we Initialize Julia in jnumpy,  then start a new python process from Julia and again initialize jnumpy in this new process, jnumpy would just throw error.~

Now if `CF_TYPY_PY_APIPTR` is invalid, we would delete this environment variable and run `init_jl()` again, so junumpy could initialize julia successfully in new process.

**Comment by songjhaha** [comment]
Duplicate of #42
